go.mod

```diff
@@ -13,10 +13,30 @@ replace (
 	go.qase.io/client => github.com/rancher/qase-go/client v0.0.0-20240308221502-c3b2635212be
 	k8s.io/api => k8s.io/api v0.32.2
 	k8s.io/apimachinery => k8s.io/apimachinery v0.32.2
+	k8s.io/apiserver => k8s.io/apiserver v0.32.2
```

Comment:
I think these changes should be relatively harmless, but I would like this to be tested by various tests first.